src/main/java/pl/wkos/homework24/model/Product.java

```diff
@@ -0,0 +1,32 @@
+package pl.wkos.homework24.model;
+
+public class Product {
+    private int productId = 0;
```

Comment:
Nie wiem czy jest to potrzebne w tym momencie - nie widzę żebyś gdziekolwiek z tego korzystał

Reply:
To miało być przygotowanie do wprowadzenia identyfikatora produktu, który miałby być zwiększany automatycznie po dodaniu nowego elementu.

Reply:
Ale nie zdążyłem.